### PR TITLE
fix: Async connection killing

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Async connection killing

### CODE REVIEW

File: lib/realtime/helpers.ex [hunk 1]
```diff
@@ -313,9 +313,13 @@ defmodule Realtime.Helpers do
   """
   @spec kill_connections_to_tenant_id_in_all_nodes(String.t(), atom()) :: :ok
   def kill_connections_to_tenant_id_in_all_nodes(tenant_id, reason) do
-    Enum.each(Node.list(), fn node ->
-      :erpc.call(node, __MODULE__, :kill_connections_to_tenant_id, [tenant_id, reason], 5000)
-    end)
+    Task.async_stream(
+      Node.list(),
```
Comment (abc3) [inline lib/realtime/helpers.ex:317]: I guess it will be invoked on all other nodes, but not on the current one, correct?

Comment (filipecabaco) [reply lib/realtime/helpers.ex:317]: you are right, I need to add node() to this list, great catch thank you

File: lib/realtime/helpers.ex [hunk 2]
```diff
@@ -314,7 +314,7 @@ defmodule Realtime.Helpers do
   @spec kill_connections_to_tenant_id_in_all_nodes(String.t(), atom()) :: :ok
   def kill_connections_to_tenant_id_in_all_nodes(tenant_id, reason) do
     Task.async_stream(
-      Node.list(),
+      Node.list() ++ [node()],
```
Comment (abc3) [inline lib/realtime/helpers.ex:317]: nitpick:
```suggestion
      [node() | Node.list()],
```
but list concatenation would also be fine for that case

**Review by abc3 — APPROVED** [review]

**Pull request merged**